src/modules/35-user-profile/navigation/UserNav/__tests__/UserNav.test.tsx

```diff
@@ -39,4 +40,46 @@ describe('User Profile Page', () => {
       )
     ).toBeTruthy()
   })
+
+  test('Logout with no return url', async () => {
+    mockLogout = jest.fn().mockReturnValue({ logoutUrl: undefined })
+    const { getByTestId } = render(
+      <TestWrapper path="/account/:accountId" pathParams={{ accountId: 'testAcc' }}>
+        <UserNav />
+      </TestWrapper>
+    )
+    const logoutBtn = getByTestId('signout-link')
+    act(() => {
+      fireEvent.click(logoutBtn)
+    })
+
+    expect(mockLogout).toBeCalled()
+    await waitFor(() =>
+      expect(getByTestId('location').innerHTML).toEqual('/redirect?returnUrl=%2F%23%2Flogin%3Faction%3Dsignout')
+    )
```

Comment:
Might as well put the `expect` in the `waitFor` incase there's a race condition.
```suggestion
    await waitFor(() => {
      expect(mockLogout).toBeCalled()
      expect(getByTestId('location').innerHTML).toEqual('/redirect?returnUrl=%2F%23%2Flogin%3Faction%3Dsignout')
    })
```

Reply:
afaik you can only put one `expect` in `waitFor`

Reply:
You can put as many as you like. It'll retry all assertions until the all pass (or it times out)

Reply:
ah okay. i messed up the syntax, thats why it wasnt working. will keep it in mind for future 👍